engines/wintermute/base/base_keyboard_state.cpp

```diff
@@ -337,8 +348,141 @@ enum VKeyCodes {
     //TODO: shift, ctrl, menu, etc...
 };
 
+#ifdef ENABLE_FOXTAIL
+enum FoxtailVKeyCodes {
+	ftVkBack       = 8,
+	ftVkTab        = 9,
+	ftVkClear      = 1073741980,
+	ftVkReturn     = 13,
+	ftVkPause      = 1073741896,
+	ftVkCapital    = 1073741881,
+	ftVkEscape     = 27,
+	ftVkSpace      = 32, //printable
+
+	ftVkPrior      = 1073741899,
+	ftVkNext       = 1073741902,
+	ftVkEnd        = 1073741901,
+	ftVkHome       = 1073741898,
+	ftVkLeft       = 1073741904,
+	ftVkUp         = 1073741906,
+	ftVkRight      = 1073741903,
+	ftVkDown       = 1073741905,
+	ftVkPrint      = 1073741894,
+	ftVkInsert     = 1073741897,
+	ftVkDelete     = 127,
+
+	ftVkF1         = 1073741882,
+	ftVkF2         = 1073741883,
+	ftVkF3         = 1073741884,
+	ftVkF4         = 1073741885,
+	ftVkF5         = 1073741886,
+	ftVkF6         = 1073741887,
+	ftVkF7         = 1073741888,
+	ftVkF8         = 1073741889,
+	ftVkF9         = 1073741890,
+	ftVkF10        = 1073741891,
+	ftVkF11        = 1073741892,
+	ftVkF12        = 1073741893,
+
+	ftVkNumLock    = 1073741907,
+	ftVkScroll     = 1073741895
+};
+#endif
+
 //////////////////////////////////////////////////////////////////////////
 uint32 BaseKeyboardState::keyCodeToVKey(Common::Event *event) {
+#ifdef ENABLE_FOXTAIL
```

Comment:
See comment below about separate functions. Can probably get away with a single set of #ifdef's around the implementations too.

Reply:
Also done in fd3846e.